roles/keystone/handlers/main.yml

```diff
@@ -2,3 +2,7 @@
 - name: restart keystone services
   service: name=keystone state=restarted must_exist=false
   when: restart|default('True')
+
+- name: restart shibboleth
+  service: name=shibd state=restarted must_exist=false
```

Comment:
Is there a scenario in which we would notify to restart shibboleth, but the service wouldn't exist? (and actually the above handler mistakenly allows the service to not exist too)